dhcpv6/option_ntp_server.go

```diff
@@ -0,0 +1,47 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/u-root/uio/uio"
+)
+
+// OptNTPServer returns an NTP Server option as defined by RFC 3646.
+func OptNTPServer(ntpServers ...net.IP) Option {
+	return &optNTPServer{NTPServers: ntpServers}
+}
+
+type optNTPServer struct {
+	NTPServers []net.IP
+}
+
+// Code returns the option code
+func (op *optNTPServer) Code() OptionCode {
+	return OptionNTPServer
+}
+
+// ToBytes returns the option serialized to bytes.
+func (op *optNTPServer) ToBytes() []byte {
+	buf := uio.NewBigEndianBuffer(nil)
+	for _, ns := range op.NTPServers {
+		buf.WriteBytes(ns.To16())
+	}
+	return buf.Data()
+}
+
+func (op *optNTPServer) String() string {
+	return fmt.Sprintf("NTP: %v", op.NTPServers)
+}
+
+// parseOptNTPServer builds an optNTPServer structure
+// from a sequence of bytes. The input data does not include option code and length
+// bytes.
+func parseOptNTPServer(data []byte) (*optNTPServer, error) {
```

Comment:
Is this following the proper option structure as defined in the RFC?
https://datatracker.ietf.org/doc/html/rfc5908

Reply:
no it's not. I noticed right after putting the PR up and commented in the internal chat, but the wire format is wrong and I have to adjust these functions